src/effects/VST3/VST3Effect.h

```diff
@@ -38,6 +38,12 @@ namespace Steinberg
 
 class ParameterChangesProvider;
 
+
+struct VST3EffectSettings
+{
+   std::unordered_map<Steinberg::Vst::ParamID, Steinberg::Vst::ParamValue> mValues;
```

Comment:
I need to add `#include <unordered_map>` to build it

Reply:
now added in commit `created ForEachParameters and used in 3 places`